eng/helix/Helix.proj

```diff
@@ -0,0 +1,164 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk"
+         DefaultTargets="Test">
+
+  <PropertyGroup>
+    <HelixConfiguration>$(Configuration)</HelixConfiguration>
+    <HelixArchitecture>$(BuildArch)</HelixArchitecture>
+    <UseOpenQueues>true</UseOpenQueues>
+    <UseOpenQueues Condition="'$(HelixAccessToken)' != ''">false</UseOpenQueues>
+    <!-- Ensure dotnet correlation payloads match the tested RID -->
+    <DotNetCliRuntime>$(PackageRid)</DotNetCliRuntime>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(UseOpenQueues)' == 'true'">
+    <QueueSuffix>.open</QueueSuffix>
+    <!-- Open queues require a creator to be set. -->
+    <Creator Condition=" '$(USERNAME)' != '' ">$(USERNAME)</Creator>
```

Comment:
Just a quick sanity check (I don't have enough context here to know) - is USERNAME or USER guaranteed to be non-empty (if not, based on the comment I'm assuming this wouldn't work), and USER is prioritized over USERNAME if both are present (or is it only one-or-the-other)?

Reply:
The `USERNAME` property comes from the Windows environment variable `USERNAME` and the `USER` property comes from the non-Windows environment variable `USER`. While these aren't explicitly mutually exclusive, they typically are and one of them should be set regardless of which environment this is running.